src/hotspot/share/opto/superword.cpp

```diff
@@ -2461,7 +2401,7 @@ bool SuperWord::output() {
 
   for (int i = 0; i < body().length(); i++) {
     Node* n = body().at(i);
-    Node_List* p = my_pack(n);
+    Node_List* p = _packset.pack(n);
```

Comment:
Hmm. I tried it, but then we often also have a variable `pack`. So I now changed the `pack(n)` to `get_pack(n)`. I think that is better anyway, it suggests that we "get" something, rather than "pack" something.